lib/superstatic.js

```diff
@@ -19,6 +19,17 @@ var loadConfigFile = require('./loaders/config-file');
 
 var CWD = process.cwd();
 
+  // shink-ray is a fork of compression which supports advanced compression encodings but not ES5.
+function initializeCompressor(options) {
+  var compressor;
```

Comment:
It was my understanding that the `superstatic()` function exported by `superstatic.js` was called only once on app initialization. `initializeCompressor()` wouldn't be called more than once, would it?